src/index.css

```diff
@@ -0,0 +1,348 @@
+}
+
+.pokemon-item img {
+  max-width: 100%;
+}
+
+#pokemon-counter {
+  margin-bottom: 50px;
+}
+
+.comment-pop-up div img {
+  width: 35vw;
+}
+
+.navbar .logo img {
+  height: 6vw;
+}
+
+.navbar nav ul {
+  display: flex;
+  list-style: none;
+}
+
+.comment-pop-up li {
+  list-style: none;
+}
+
+.comment-pop-up-details ul li {
+  width: 40%;
+}
+
+.navbar nav ul li {
+  margin: 0 1vw;
+}
+
+#comment-list li {
+  font-size: 0.8rem;
+  font-family: cursive;
+}
+
+.navbar nav ul li a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.lists a:hover {
+  color: rgb(36, 94, 180);
+}
+
+.lists a.active {
+  color: rgb(36, 94, 180);
+}
+
+.navbar nav ul li a:hover {
+  color: rgb(36, 94, 180);
+}
+
+.navbar nav ul li a.active {
+  color: rgb(36, 94, 180);
+}
+
+.logo {
+  background-image: url('./assets/logo/pokemon-logo.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 6vw;
+  height: 6vw;
+  background-position: center;
+}
+
+.header {
+  height: 6vw;
+  display: flex;
+  align-items: center;
+}
+
+footer {
+  height: 5vw;
+  display: flex;
+  align-items: center;
+  border: 2px solid #000;
+  box-shadow:
+    3px 3px 0 2px #000,
+    -3px -3px 0 2px #c0c0c0,
+    3px 3px 5px 4px #a9a9a9 inset,
+    -3px -3px 5px 4px #fff inset;
+  padding: 0 1vw;
+  position: sticky;
+  background-color: #fff;
+  bottom: 0;
+  z-index: 1000;
+}
+
+.footer {
+  font-size: 1vw;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 90vw;
+  padding: 0 1vw;
+  position: relative;
+  background-color: #fff;
+  height: 5vw;
+  bottom: 0;
+  z-index: 1000;
+}
+
+/* ---------- POKEMON LIST ---------- */
+
+.pokemon-list {
+  flex-shrink: 0;
+  display: flex;
+  margin-top: 0.1vw;
+  margin-bottom: 5vw;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-content: center;
+  height: 100vh;
+  padding-top: 7.5vw; /* Added to account for the navbar */
+}
+
+.pokemon-item {
+  flex-basis: calc(30% - 2em);
+  margin: 1em;
+  padding: 2em;
+  text-align: center;
+  box-sizing: border-box;
+  height: 50vh;
+  width: 20vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  justify-content: space-between; /* Added to space out the elements */
+  background-color: #fff;
+  border: 2px solid #000;
+  box-shadow:
+    3px 3px 0 2px #000,
+    -3px -3px 0 2px #c0c0c0,
+    3px 3px 5px 4px #a9a9a9 inset,
+    -3px -3px 5px 4px #fff inset;
+}
+
+.comment-button {
+  padding: 1em;
+  background-color: #fff;
+  border: 2px solid #000;
+  box-shadow:
+    3px 3px 0 2px #000,
+    -3px -3px 0 2px #c0c0c0,
+    3px 3px 5px 4px #a9a9a9 inset,
+    -3px -3px 5px 4px #fff inset;
+  transition: 0.3s;
+  text-decoration: none;
+  text-align: center;
+}
+
+.comment-button:hover {
+  background-color: #c0c0c0;
+}
+
+.comment-button:active {
+  box-shadow:
+    -3px -3px 0 2px #000,
+    3px 3px 0 2px #c0c0c0,
+    -3px -3px 5px 4px #a9a9a9 inset,
```

Comment:
- [ ] Kindly remove the grey placement that blocks the comments button from been visible 👍 

### View:
![image](https://github.com/rudicarrilloypr/kanban-capstone-js-module2/assets/50990925/5643b819-dcdc-43f4-aefc-0d3b9ae6b96a)

Reply:
that grey block is of the deployment platform we used to deploy the project....
<img width="1359" alt="Captura de pantalla 2023-07-07 a la(s) 7 07 50" src="https://github.com/rudicarrilloypr/kanban-capstone-js-module2/assets/125106965/2d9f55a7-9f8c-4bfa-9b29-ea42ff67d6c7">